hack/choco/update-choco-package.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+ 
+# Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+ 
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+ 
+version="${1:?TCE version argument empty. Example usage: ./hack/choco/update-choco-package.sh v0.10.0}"
+: "${GITHUB_TOKEN:?GITHUB_TOKEN is not set}"
+ 
+ 
+temp_dir=$(mktemp -d)
+ 
+pushd "${temp_dir}"
+
+TCE_REPO="https://github.com/vmware-tanzu/community-edition/" 
+TCE_REPO_RELEASES_URL="https://github.com/vmware-tanzu/community-edition/releases"
+TCE_WINDOWS_TAR_BALL_FILE="tce-darwin-amd64-${version}.tar.gz"
+TCE_CHECKSUMS_FILE="tce-checksums.txt"
+ 
+echo "Checking if the necessary files exist for the TCE ${version} release"
+ 
+curl -f -I -L \
```

Comment:
For both of these, we don't care about the contents of the files, just that they exist. I think this and line 32 could be updated to be:

```sh
wget --spider -q "${URL}" || {
```

Reply:
Agree I will update this.